src/preact.js

```diff
@@ -9,11 +9,16 @@ function createRef() {
 	return {};
 }
 
+function forwardRef(render) {
+	return { $$typeof: Symbol.for('react.forward_ref'), render };
```

Comment:
No need to match React's format here - we don't use Symbol for browser compat and performance reasons. I think this could be:


```
// in src/constants.js
const FORWARD_REF = function() {};

function forwardRef(render) {
  return h(FORWARD_REF, null, render);
}

// processing step (from below):
	if (vnode.nodeName===FORWARD_REF) {
		const props = getNodeProps(vnode)
		vnode = vnode.children[0](props, vnode.ref);

```